tests/tools/saunafsXX.sh

```diff
@@ -141,7 +140,9 @@ assert_saunafsXX_services_count_equals() {
 }
 
 assert_no_saunafsXX_services_active() {
-	assert_saunafsXX_services_count_equals 0 0 0
+	assert_equals 0 "$(saunafs_admin_master info | grep "${SAUNAFSXX_TAG}" | wc -l)"
```

Comment:
@uristdwarf why having it here in 3 lines is better then in one line ?

Reply:
is it because there was `saunafs_old_admin_master` inside of `assert_saunafsXX_services_count_equals`

can you explain logic here?

why there are two of them?
`saunafs_old_admin_master` & `saunafs_admin_master`

old is before upgrade...also called XX and other is after upgrade ?

Reply:
I am bit lost... something is wrong about this XX in name that it is confusing... maybe we should use 'previous' and 'current 'or 'old' and 'new' instead of 'XX' and '' 


however this is separate refactoring... not related with this PR directly...
So I will accept it.

Reply:
Yes, XX is the "old" version, or rather the previous version from the current. Why such an awkward name was chosen is a mystery to me.

> is it because there was saunafs_old_admin_master inside of assert_saunafsXX_services_count_equals
can you explain logic here?
why there are two of them?
saunafs_old_admin_master & saunafs_admin_master
old is before upgrade...also called XX and other is after upgrade ?

The problem occurred when all the old services where stopped and replaced with new ones. The metadata parallel dumping feature modified the info command API with master, which means older admin executables no longer work with newer masters/chunkservers etc. The solution here was to stop using the old version and instead use the newer one instead to check it.

This is actually something of a question of mine, should we support using older versions of executables to communicate with newer services? And which executables/services exactly?

Reply:
Actually, I believe the newer version cannot communicated either